cmd/ceremony/README.md

```diff
@@ -129,6 +129,8 @@ certificate-profile:
 
 This config generates an intermediate certificate signed by a key in the HSM, identified by the object label `root signing key` and the object ID `ffff`. The subject key used is taken from `/home/user/intermediate-signing-pub.pem` and the issuer is `/home/user/root-cert.pem`, the resulting certificate is written to `/home/user/intermediate-cert.pem`.
 
+Note: Intermdiate certificates always include the extended key usages id-kp-serverAuth and id-kp-clientAuth as required by 7.1.2.2.g of the CABF Baseline Requirements.
```

Comment:
BRs only require serverAuth:

> For Subordinate CA Certificates that will be used to issue TLS certificates, the value
id-kp-serverAuth [RFC5280] MUST be present. The value id-kp-clientAuth
[RFC5280] MAY be present. 

We should describe that here, e.g. "serverAuth is required, but we currently always include clientAuth because we include clientAuth on our EE certs; we may change that in the future."

Also typoe on "Intermdiate"